source/more-info/local-media/setup-media.markdown

```diff
@@ -9,7 +9,7 @@ find you local media.
 ## Home Assistant OS and Supervised
 
 No action is needed from your end to set it up. Home Assistant will
-automatically use the "media" folder that is provided on these systems.
+automatically use the "media" folder that is provided on these systems (e.g.: `/usr/share/hassio/media/`).
```

Comment:
This is incorrect. It is `/media`.

However, this document is about setting it up, which you don't have too, as that is automatically done.